lib/flutter_redux.dart

```diff
@@ -535,21 +521,21 @@ class _StoreStreamListenerState<S, ViewModel>
         // latestValue. Important: This must be done after all other optional
         // transformations, such as ignoreChange.
         .transform(StreamTransformer.fromHandlers(
-            handleData: _handleChange, handleError: _handleError));
+          handleData: _handleChange,
+          handleError: _handleError,
+        ));
   }
 
   void _handleChange(ViewModel vm, EventSink<ViewModel> sink) {
     _latestError = null;
 
-    if (widget.onWillChange != null) {
-      widget.onWillChange(_latestValue, vm);
-    }
+    widget.onWillChange?.call(_latestValue!, vm);
 
     _latestValue = vm;
 
     if (widget.onDidChange != null) {
-      WidgetsBinding.instance.addPostFrameCallback((_) {
-        widget.onDidChange(_latestValue);
+      WidgetsBinding.instance?.addPostFrameCallback((_) {
```

Comment:
I dropped my prepared PR in favor of yours - I learned a lot from your code.
May I ask if this call is really null safe?
I'd phrase it as
```
WidgetsBinding.instance?.addPostFrameCallback?.call
```

Reply:
Yes, it will not call addPostFrameCallback if it is null🙂